dangerfile.ts

```diff
@@ -11,6 +11,22 @@ function checkPackageLock() {
   }
 }
 
+function checkSmartContractGitModules() {
+  const updatesContracts = contains(
+    danger.github.pr.body + danger.github.pr.title,
+    "#with-contracts",
+  );
+
+  const hasContractSubmodule =
+    danger.git.modified_files.filter(p => p.includes("platform-contracts-artifacts/")).length > 0;
```

Comment:
to simplify we maybe we can use `some`? 
```suggestion
    danger.git.modified_files.some(p => p.includes("platform-contracts-artifacts/"));
```